src/test/java/net/helpscout/api/WebhookApiTest.java

```diff
@@ -0,0 +1,109 @@
+package net.helpscout.api;
+
+import lombok.SneakyThrows;
+import net.helpscout.api.model.Customer;
+import org.junit.Test;
+
+import javax.servlet.http.HttpServletRequest;
+import java.io.BufferedReader;
+import java.io.StringReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.*;
+
+/**
+ * @Author: ivan
+ * Date: 22.03.16
+ * Time: 0:33
+ */
+public class WebhookApiTest {
+
+
+    /**
+     * Real webhook request contains such kind of headers:
+     * host: youcallback.domain.com
+     * user-agent: Help Scout/Webhooks
+     * accept-encoding: UTF-8
+     * content-type: application/json
+     * x-helpscout-event: customer.created
+     * x-helpscout-signature: qALfoJFZ/WVbevIxtFYKHJ86D8o=
+     * x-forwarded-proto: http
+     * x-forwarded-port: 80
+     * x-request-start: t=1458595138602884
+     * x-client-ip: 000.000.000.000
+     * content-length: 406
+     * x-forwarded-host: youcallback.domain.com
+     * x-forwarded-server: youcallback.domain.com
+     * connection: Keep-Alive
+     *
+     * Pay attention to case of header names - according to specification
```

Comment:
What documentation do you have in mind? `getHeader()` is case insensitive : http://docs.oracle.com/javaee/1.3/api/javax/servlet/http/HttpServletRequest.html#getHeader%28java.lang.String%29. Or did you mean API documentation?